commit 4836a9af89f1b4d482e6c72ff67e36226d36434c
Author: Florian Weimer <fweimer@redhat.com>
Date:   Tue Nov 26 19:26:13 2024 +0100

    debug: Fix tst-longjmp_chk3 build failure on Hurd
    
    Explicitly include <unistd.h> for _exit and getpid.

diff --git a/debug/tst-longjmp_chk3.c b/debug/tst-longjmp_chk3.c
index dfdecca9ef8a1cf6..254cd671902ca9a0 100644
--- a/debug/tst-longjmp_chk3.c
+++ b/debug/tst-longjmp_chk3.c
@@ -20,6 +20,7 @@
 #include <signal.h>
 #include <stdio.h>
 #include <string.h>
+#include <unistd.h>
 
 #include <support/support.h>
 
